src/main/resources/static/js/main.js

```diff
@@ -70,18 +70,20 @@ jQuery(document).ready(function() {
     });
 
     function doIfUserLoggedIn(currentUser) {
-        if (currentUser != "") {
```

Comment:
Do we not need this check?

Reply:
You never enter this function unless a successful login request was made and that request can only be successful if a user has once logged in with the right credentials. This means that it must have a value if it was successful so I didn't feel it was needed to check whether it was empty or not.